packages/app/src/cli/services/build/extension.ts

```diff
@@ -175,6 +175,11 @@ export async function buildFunctionExtension(
     } else {
       await buildOtherFunction(extension, options)
     }
+
+    if (fileExistsSync(extension.outputPath)) {
```

Comment:
Javy doesn't need wasm-opt since those modules only have a very minimal handwritten set of Wasm instructions. But it shouldn't hurt to run wasm-opt on those modules.